### PR TITLE
author sumbission: add current checkbox

### PR DESCRIPTION
**Pull request by MJedr**
ref:#2309

### CODE REVIEW

**Review by michamos — COMMENTED** [review]
looks nice :+1: did you disable removal of email addresses for non-curators, like happens for `career info` (didn't see it in the code)? could you also add an explanation under `Personal Info` like we do for `Career Info`:
> Email addresses cannot be deleted. If information is incorrect, please mask it as 'Hidden' and give more details in the Comments area.

File: ui/src/submissions/authors/components/AuthorForm.jsx [hunk 1]
```diff
@@ -82,6 +87,7 @@ function AuthorForm({ values, isCatalogerLoggedIn, isUpdate }) {
             name="emails"
             label="Emails"
             emptyItem={{}}
+            allowItemDelete={isCatalogerLoggedIn || !isUpdate}
```
Comment (MJedr) [inline ui/src/submissions/authors/components/AuthorForm.jsx:90]: @michamos here's the check for deletion

File: ui/src/submissions/authors/components/AuthorForm.jsx [hunk 2]
```diff
@@ -47,6 +47,11 @@ function AuthorForm({ values, isCatalogerLoggedIn, isUpdate }) {
     <Form>
       <CollapsableForm openSections={OPEN_SECTIONS}>
         <CollapsableForm.Section header="Personal Info" key="personal_info">
+          <p>
+            Email addresses cannot be deleted. If information is incorrect,
+            please mask it as `Hidden` and give more details in the Comments
```
Comment (michamos) [inline ui/src/submissions/authors/components/AuthorForm.jsx:52]:
```suggestion
            please mark it as `Hidden` and give more details in the Comments
```

**Review by michamos — APPROVED** [review]
small typo, looks good otherwise!

**Pull request merged**